helm/vertical-pod-autoscaler-app/templates/crd-patch/ciliumnetworkpolicy.yaml

```diff
@@ -0,0 +1,24 @@
+{{ if .Values.ciliumNetworkPolicy.enabled }}
+apiVersion: cilium.io/v2
+kind: CiliumNetworkPolicy
+metadata:
+  name: vpa-annotate-job-np
+  annotations:
+    helm.sh/hook: "post-install,post-upgrade"
+    helm.sh/hook-weight: "-7"
+    helm.sh/hook-delete-policy: "before-hook-creation,hook-succeeded"
+  labels:
+    {{- include "vertical-pod-autoscaler.labels" . | nindent 4 }}
+specs:
+  - description: "Allow the Job to access the K8s API Server"
+    endpointSelector:
+      # Since we are excluding the job-name from the matched resources from cilium identity evaluation
+      # we simply can't use it, because the policy will never be matched against the job
+      # If and when this will be reverted: https://github.com/giantswarm/cilium-app/pull/80/files#diff-19d6b2fc3287f6aa81dac49278df30124546319c557f738958b6f4b1326e4867R1422
+      # we can use the job name once again
+      matchLabels: {}
```

Comment:
@Gacko found a fix here https://github.com/giantswarm/vertical-pod-autoscaler-app/commit/b02f04a8f8a739cdbf6aea632e4ef7c33f2c8b12 if you can implement it in this pr that would be awesome